New log file opened at Sun May 03 08:19:44 PDT 2015 by 29939@Thomass-MacBook-Pro.local
Taking next connection to: 59021
New client attempting to connect on port: 59022-59021
  Address = /192.168.1.4:59021
This client on port 59022 has uuid 878ebeb0-414f-468e-ade7-bbb0b4b0aecb configName activator humanReadableName Activator protocolVersion 1 featureTags Vector()
Connected Clients: activator-878ebeb0-414f-468e-ade7-bbb0b4b0aecb
Taking next connection to: 59021
New client attempting to connect on port: 59023-59021
  Address = /192.168.1.4:59021
Handshake exception on socket: 59023
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Connection reset
	at java.net.SocketInputStream.read(SocketInputStream.java:189)
	at java.net.SocketInputStream.read(SocketInputStream.java:121)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 59021
New client attempting to connect on port: 59024-59021
  Address = /192.168.1.4:59021
This client on port 59024 has uuid 6c5142ec-201e-4172-8526-b6a573dddba7 configName activator humanReadableName Activator protocolVersion 1 featureTags Vector()
Connected Clients: activator-878ebeb0-414f-468e-ade7-bbb0b4b0aecb, activator-6c5142ec-201e-4172-8526-b6a573dddba7
Taking next connection to: 59021
New client attempting to connect on port: 59032-59021
  Address = /192.168.1.4:59021
Handshake exception on socket: 59032
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:392)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 59021
New client attempting to connect on port: 59033-59021
  Address = /192.168.1.4:59021
This client on port 59033 has uuid b01fccb0-10b3-4fee-a653-d6af841919da configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-6c5142ec-201e-4172-8526-b6a573dddba7, play-fork-run-b01fccb0-10b3-4fee-a653-d6af841919da
Taking next connection to: 59021
